shared/telemetry/README.md

```diff
@@ -0,0 +1,121 @@
+# Welcome to the Telemetry Module
+
+This module is used to collect telemetry data from the node.
+
+At the moment, we are using two types of metrics:
+
+- Time series metrics: collected periodically and are stored in a database.
```

Comment:
NIT: Make `Time Series metics` bold. DItto for event series metric and other headers in bullet points below.

Reply:
Also, maybe this is clearer: "Metrics collected at regular intervals and stored in a database"